lib/charting/c3_charting.rb

```diff
@@ -66,7 +66,7 @@ def chart_names_for_select
 
   # list of themes - in options_for_select format
   def chart_themes_for_select
-    %w(Default default)
+    %w(Default)
```

Comment:
I believe the fix here should read something like:

```ruby
[["Default", "default"]]
```

which I believe was the original intent of the author (i.e. to display "Default" and to submit "default").